File: Covid-19.java
import java.util.Scanner; 
public class Main {
	public static void main(String[] args) {
		boolean Loop = true;
		Scanner sc= new Scanner(System.in);
		do {
		    System.out.println("Do you have a Fever?");
		    String ans = sc.next();
		    System.out.println("Do you have cough?");
		    String ans1 = sc.next();
		    System.out.println("Do you have difficulty breathing");
		    String ans2 = sc.next();
		    
		    
		    if (ans.equalsIgnoreCase("Yes")||
		    ans1.equalsIgnoreCase("Yes") || ans2.equalsIgnoreCase("Yes")){
		        System.out.println("You may have COVID-19 symptoms, Please consult a healthcare professional");
		        } else{
		            System.out.println("You do not exhibits significance of COVID-19 symptoms, Stay Safe!");
	} System.out.println("Do you want consult again? yes/no");
		    String answer = sc.next();
		    if (answer.equalsIgnoreCase("Yes")) {
		        } else {
		            Loop = false;
		            }
		     
		} while (Loop);
}
}

File: Covid19.py
loop = True

while loop:
    q1 = input("Do you have a fever?: ")
    q2 = input("Do you have cough?: ")
    q3 = input("Do you have difficulty breathing?: ")

    if (q1 == q1.lower("Yes")) or (q2 == q2.lower("Yes") or (q3 == q3.lower("Yes"):
        print("You may have COVID-19 symptoms. Please consult healthcare professional")
    else:
        print("You do not exhibit significant COVID-19 symptoms. Stay Safe!")
    answer = input("Do you want to check again?")
    if answer == 'Yes' or 'No':
        loop = True
    else:
        loop = False